### ISSUE TITLE
Multiple arguments for dimension

### ISSUE THREAD

**Issue by IainNZ** [JuliaStats/Distributions.jl#377]
```julia
julia> using Distributions

julia> rand(Uniform(0,1),10,2)
ERROR: `rand` has no method matching rand(::Uniform, ::Int64, ::Int64)
```

Surprised me this wasn't implemented, but I'm guessing there is a reason? Is it ambiguous in some contexts what is desired? I feel like for univariate distributions its unambiguous.

**Comment by johnmyleswhite** [comment]
I'm pretty sure this used to work.

**Comment by johnmyleswhite** [comment]
https://github.com/JuliaStats/Distributions.jl/blob/master/src/univariates.jl#L64

I suspect making a variant with `Dims...` is sufficient.

**Comment by IainNZ** [comment]
Yeah, its seems simple enough that I wondered if there was a choice made at some point to not do it.

**Comment by johnmyleswhite** [comment]
I don't remember such a choice.

**Comment by lindahua** [comment]
The Julia base supports such:
```julia
julia> rand(1:10, 3, 4)
3x4 Array{Int64,2}:
 9  6  5   3
 4  3  7  10
 9  8  9   1
```
To be consistent, we should support this too.

**Comment by lindahua** [comment]
The support of ``rand(d, dims...)`` has been implemented by 60e536e.
